assemble/bin/bootstrap_config.sh

```diff
@@ -24,7 +24,7 @@ where options include (long options not available on all platforms):
     -n, --native     Configure to use native libraries
     -j, --jvm        Configure to use the jvm
     -o, --overwrite  Overwrite the default config directory
-    -v, --version    Specify the Apache Hadoop version supported versions: '1' '2'
+    -v, --version    Specify the Apache Hadoop version supported versions: '1' '2' '3'
```

Comment:
Ha! We don't actually support version 1 anymore. Probably need to remove that from the list.